hbase-server/src/test/java/org/apache/hadoop/hbase/coprocessor/TestRegionCoprocessorHost.java

```diff
@@ -79,19 +81,36 @@
 
   @Before
   public void setup() throws IOException {
+    init(null);
+  }
+
+  public void init(Boolean flag) throws IOException {
```

Comment:
Let's use private here? I do not think this method needs to be called in other classes?